Fix wrong description when switching project CRS (#9825)
qgis/QGIS-Documentation: docs/user_manual/introduction/qgis_configuration.rst

    New International 1967, WGS 84...).
* the :guilabel:`Units for distance measurements` for length and perimeter,
  and the :guilabel:`Units for area measurements`.
  These settings default to their corresponding :ref:`global options <global_measure_tool>`
  and can be manually changed for the current project.

> .. note::

Reviewer: ```suggestion
.. tip::
```
Let's use it as a tip

>    To ensure measurement units (for distance and area) reflect the units of the current project's CRS,
>    select the corresponding unit from the drop-down menu.
>    If you are unsure which unit your desired CRS uses for its coordinates,
>    you can choose :guilabel:`Map Units`, which will use the units of the current project's CRS.

Reviewer: If I'm not mistaken, the idea is "you change your crs, you do nothing but you are sure that your units are aligned with it". The only option that ensures this scenario is "map units". So let's directly suggest it.
```suggestion
   To ensure measurement units (for distance and area) always reflect the units of the current project's CRS,
   select :guilabel:`Map Units` for these options.
```
